app/front/presenters/ConcertPresenter.php

```diff
@@ -22,4 +22,9 @@ public function actionDefault () {
         }
     }
 
+    public function actionDetail ( $id ) {
+        if ( ! $this -> template -> concert = $this -> concerts -> item ( $id ) )
```

Comment:
When concert does not exist we should return 404 and no redirect. [@see](https://github.com/bigbandbiskupska/bigbandbiskupska/blob/master/app/front/presenters/BasePresenter.php#L20) this method